cpp/ql/src/experimental/Best Practices/WrongUintAccess.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @id cpp/wrong-uint-access
+ * @name Wrong Uint
+ * @descripion Acess an array of size lower than 256 with a uint16.
+ * @kind problem
+ * @problem.severity recommendation
+ * @tags efficiency
+ */
+
+import cpp
+import semmle.code.cpp.controlflow.SSA
```

Comment:
I don't think you're actually using the SSA library for anything, are you?

Reply:
Done